spec/suites/map/MapSpec.js

```diff
@@ -2063,4 +2063,24 @@ describe("Map", function () {
 			expect(p.y).to.be.equal(200);
 		});
 	});
+
+	describe("#panBy", function () {
+
+		it("throws if map is not set befor", function () {
+			expect(function () {
+				map.panBy();
+			}).to.throwError();
```

Comment:
The exception is throw because `panBy` has no argument, not because `setView` call is missing. Please fix